web/server.js

```diff
@@ -14,8 +15,12 @@ module.exports = function(config) {
       coinbase_secret: config.coinbase_secret
     }
   });
+  server.connection({
+    host: config.host,
+    port: config.port
+  });
 
-  server.pack.register(require("hapi-auth-bearer-token"), function(err) {});
+  server.register(require("hapi-auth-bearer-token"), function(err) {});
```

Comment:
I originally had an if clause here, but the problem is that I couldn't get 100% test coverage for this file with it, since this module basically never throws an error on registration. I'm not sure how to do that...